terraform/Subnets.tf

```diff
@@ -0,0 +1,37 @@
+
+variable "cidrBlock" {
+type = list
+default = [
+"10.0.0.0/24",
+"10.0.16.0/24",
+"10.0.32.0/24",
+"10.0.48.0/24",
+"10.0.64.0/24",
+"10.0.80.0/24",
+"10.0.96.0/24",
+"10.0.112.0/24",
+"10.0.128.0/24",
+"10.0.144.0/24"
+]
+}
```

Comment:
I would create a list of complex objects like
```
{
  department: "managment_a",
  cidrBlock: "10.0.0.0/24",
availabilityZone: "us-west-2a"
}
```

and than loop over this objects